Services/DataBaseService/DbModels/DbUserCredential.cs

```diff
@@ -9,6 +9,7 @@ public class DbUserCredential
         public Guid Id { get; set; }
         public string Email { get; set; }
         public string PasswordHash { get; set; }
+        public bool Exist { get; set; }
```

Comment:
The word exist isn't understandable. I think that it means that user hadn't delete himself. But it is dbmodel. This User always exist in database